service.libraryautoupdate/resources/language/Afrikaans/strings.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" standalone="yes"?>
```

Comment:
it would be nice if you update the strings to the new .po format at some point. You can do that later though